.travis.yml

```diff
@@ -1,10 +1,21 @@
 sudo: false
-
 language: ruby
-
 rvm:
-  - 2.3.3
-  - 2.4.0
-
+- 2.4.5
 cache: bundler
 script: bundle exec middleman build
+deploy:
+  provider: s3
+  access_key_id: $AWS_ACCESS_KEY_ID
```

Comment:
I'm just using Travis encrypted variables in "Settings"

![image](https://user-images.githubusercontent.com/8136030/56711559-23275b00-66e0-11e9-8c75-dc1365f3288e.png)

Reply:
ah cool, I think I actually like this better than using travis's in-file secrets